copulas/utils.py

```diff
@@ -19,6 +19,7 @@
 RAW_EXT = '.raw.csv'
 SYNTH_EXT = '.synth.csv'
 TRANS_EXT = '.trans.csv'
+EPSILON = np.finfo(np.float32).eps
```

Comment:
I think most part of this file is not being used anymore. The classes implemented in this file is used for the SDV project. Would you mind removing them?

Reply:
I had planned doing that along issue #11. After move the test data functions out of here, I'll delete this file.